modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/_clay_extend.scss

```diff
@@ -0,0 +1,3 @@
+.hide {
+	display: none !important;
+}
```

Comment:
Just to understand the point of view, why this class is considered a clay extension?

Reply:
I see the class has been already moved to clay! :-D